src/engine/strat_engine/blockdev.rs

```diff
@@ -207,7 +207,7 @@ pub fn initialize(pool_uuid: &PoolUuid,
 }
 
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Looks like we can avoid the clone by: 

        let meta_dev;
        let data_dev;
        let length;
        {
            let mut data_devs = Vec::from_iter(bds.values());
            let metadata_blockdev = data_devs.remove(0);

            data_dev = try!(LinearDev::new(&format!("stratis_{}_data", name), dm, &data_devs));
            length = try!(data_dev.size()).sectors();

            let meta_blockdevs = vec![metadata_blockdev];
            meta_dev = try!(LinearDev::new(&format!("stratis_{}_meta", name), dm, &meta_blockdevs));
        }
        // TODO Fix hard coded data blocksize and low water mark.
        let thinpool_dev = try!(ThinPoolDev::new(&format!("stratis_{}_thinpool", name),
                                                 dm,
                                                 length,
                                                 Sectors(1024),
                                                 DataBlocks(256000),
                                                 meta_dev,
                                                 data_dev));